test/e2e/framework/util.go

```diff
@@ -4665,7 +4665,18 @@ func CheckConnectivityToHost(f *Framework, nodeName, podName, host string, timeo
 		return err
 	}
 	defer podClient.Delete(podName, nil)
-	return WaitForPodSuccessInNamespace(f.Client, podName, contName, f.Namespace.Name)
+	err = WaitForPodSuccessInNamespace(f.Client, podName, contName, f.Namespace.Name)
+
+	if err != nil {
+		logs, logErr := GetPodLogs(f.Client, f.Namespace.Name, pod.Name, contName)
+		if logErr != nil {
+			Logf("Warning: Failed to get logs from pod %q container %q: %v", pod.Name, contName, logErr)
+		} else {
+			Logf("wget logs:\n%s", logs)
```

Comment:
more serious small suggestion: maybe include the namespace, pod, and container names in this message?